src/main/java/com/vinylteam/vinyl/dao/RecoveryPasswordDao.java

```diff
@@ -0,0 +1,15 @@
+package com.vinylteam.vinyl.dao;
+
+import com.vinylteam.vinyl.entity.RecoveryToken;
+
+import java.util.Optional;
+
+public interface RecoveryPasswordDao {
+
+    boolean addRecoveryUserToken(RecoveryToken recoveryToken);
+
+    Optional<RecoveryToken> getByRecoveryToken(String token);
+
+    boolean removeRecoveryUserToken(String token);
```

Comment:
addRecoveryToken, findByToken, removeByToken.

Reply:
I've already changed to add, findByToken, deleteById